gcp/workers/importer/importer.py

```diff
@@ -123,7 +123,8 @@ def replace_importer_log(client: storage.Client, source_name: str,
                          bucket_name: str, import_failure_logs: List[str]):
   """Replace the public importer logs with the new one."""
   bucket: storage.Bucket = client.bucket(bucket_name)
-  upload_string = '--- ' + datetime.datetime.utcnow().isoformat() + ' ---\n'
+  upload_string = '--- ' + datetime.datetime.now(
+      datetime.timezone.utc).isoformat() + ' ---\n'
```

Comment:
nit: I think
```suggestion
  upload_string = f'--- {datetime.datetime.now(datetime.UTC).isoformat()} ---\n'
```
will just barely fit on one line and be easier to read